### DISCUSSION TITLE
readme 里面给的本地编译命令无法使用，我自己官方文档找到了命令

### DISCUSSION THREAD

**Discussion by Laifang** [LyleMi/Learn-Web-Hacking#23] [General]
1. git clone https://ghproxy.com/https://github.com/LyleMi/Learn-Web-Hacking.git
2. cd Learn-Web-Hacking
3. pip install sphinx sphinx-rtd-theme
4. `sphinx-build -b html source/ build/html`

**Comment by LyleMi** [comment]
感谢提交，仓库中提供的同样为标准makefile文件。
考虑可能是由于环境导致复现情况makefile不可用的情况，方便的话请提交详细的报错信息。
